youtube-gpt-client/src/components/mainContent/index.tsx

```diff
@@ -9,7 +9,7 @@ const MainContent = ({ children }: PropsWithChildren) => {
       backgroundColor="#fff"
       borderRadius="20px 20px 0 0"
       minHeight="calc(100vh - 60px)"
-      padding="55px 50px"
+      padding={['20px 10px', '55px 50px']}
```

Comment:
```suggestion
      padding="20px 10px 55px 50px"
```

Reply:
Same here, we are using different values to each device 20px and 10px to mobile and 55px and 50px to desktop